app/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-filter==0.11.0
 django-extensions==1.6.1
 django-cors-headers==1.0.0
 google-api-python-client==1.5.0
+oauth2client==2.0.1
```

Comment:
No worries! I'll go ahead and open up a separate PR so the fix isn't tied to this larger feature.